src/Uno.UI.RuntimeTests/Tests/Windows_UI_Xaml_Controls/Given_ListViewBase.cs

```diff
@@ -2778,330 +2729,356 @@ public async Task When_Smooth_Scrolling()
 #endif
 
 		[TestMethod]
-		public async Task When_SelectionMode_Is_Multiple()
```

Comment:
Why did you removed this test? Is it expected?

Reply:
good catch, no. must be copy-pasted over